democracy/views/hearing.py

```diff
@@ -62,23 +44,40 @@ def get_sections(self, hearing):
         serializer.bind('sections', self)  # this is needed to get context in the serializer
         return serializer.to_representation(queryset)
 
+    def get_main_image(self, hearing):
```

Comment:
Don't `SectionImage`s have `ordering = ('ordering',)` or similar in their Metas? That'd mean `.first()` implicitly grabs the first-ordered image from the intro section...